lib/vttcue.js

```diff
@@ -258,5 +304,5 @@
     return WebVTT.convertCueToDOMTree(window, this.text);
   };
 
-  root.VTTCue = root.VTTCue || VTTCue;
+  root.VTTCue = VTTCue || root.VTTCue;
```

Comment:
I should probably change this back. Though, it's being done in https://github.com/gkatsev/vtt.js/pull/1/files which I would like to get pulled in.